test/common/many-types.xml

```diff
@@ -27,6 +42,11 @@
         <arg type="ay" name="Param2" direction="out"></arg>
     </method>
 
+    <method name="TestStructArray">
```

Comment:
Would be nice to just see a plain simple struct here as well (`(ss)` for instance).

Reply:
I'll add one!